src/app/home/home.hbs

```diff
@@ -0,0 +1,33 @@
+  <section class="home-container">
+    <div class="slideshow-container">
+        {{#each banners.banners}}
+        <div class="bannerSlides fade">
+                <img src="{{bannerImageUrl}}" style="width:100%" alt="{{bannerImageAlt}}">
+        </div>
+        {{/each}}
+      <a class="prev">&#10094;</a>
+      <a class="next">&#10095;</a>
+      <br>
+      <div style="text-align:center;">
+        {{#each banners.banners}}
+        <span class="dot dot-{{@index}}"></span>
```

Comment:
Its clickable element, why its a <span> and not a button??

Reply:
Updated.